src/main/java/mr/springbootapi/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,34 @@
+package mr.springbootapi.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@ControllerAdvice
+public class GlobalExceptionHandler {
+
+    @ExceptionHandler(ItemNotFoundException.class)
```

Comment:
add a comment explaining what it does
```suggestion
    @ExceptionHandler(ItemNotFoundException.class) //It means that this method will be called to handle when ItemNotFoundException is thrown
```